package-lock.json

```diff
@@ -1,6135 +1,5 @@
 {
-  "name": "@proximacapital/eslint-config",
+  "name": "@proxima-oss/eslint-config",
   "version": "4.0.2",
-  "lockfileVersion": 2,
```

Comment:
Re-install using node `16.6.1` which is defined in `.nvmrc`. I'll handle this cause I want to update package.json